### PR TITLE
Add interface variables to nc output in standalone homme

### PR DESCRIPTION
**Pull request by oksanaguba**
Adds native and interpolated output of interface quantities in homme, in particular, for w and mu (dpnh/dphy). Fixes std output for interface quantities to have min/max calculated on interfaces, not averaged level values.
 
[bfb], affects only standalone homme.

### CODE REVIEW

**Comment by oksanaguba** [comment]
sanity check -- from plotting both wnlev and wnlevp they look the same (up to the bottom point).

**Comment by oksanaguba** [comment]
By some reason baroCamMoistSL ran on anvil without issues in this branch and commit it branched off.

**Comment by oksanaguba** [comment]
SW tests were fixed, this is ready.

**Comment by oksanaguba** [comment]
Reran previous interp and native output with the newest commit and files are identical (folder /home/onguba/as/run-test-wmovie-pr/movies ).

File: components/homme/src/theta-l/element_ops.F90 [hunk 1]
```diff
@@ -132,6 +135,35 @@ recursive subroutine get_field(elem,name,field,hvcoord,nt,ntQ)
...
+         call abortmp('ERROR: get_field_i is not supported for w in theta HY')
+      else
+         field = elem%state%w_i(:,:,1:nlevp,nt)
+      endif
+    case('geopotential');
+      !is phinh_i set for HY runs or should here be an abort?
+      field = elem%state%phinh_i(:,:,1:nlevp,nt)
+    case('mu');
+      call get_dpnh_dp_i(elem,field,hvcoord,nt)
+    case default
```
[the remark this answers is not in the document]
Comment (oksanaguba) [reply components/homme/src/theta-l/element_ops.F90]: Will do. I removed w from interp_output but i will put it back. geo and geo_i are already there.

File: components/homme/src/theta-l/element_state.F90 [hunk 2]
```diff
@@ -80,6 +79,8 @@ module element_state
     real (kind=real_kind) :: FQps(np,np)                   ! forcing of FQ on ps_v
 
     real (kind=real_kind) :: gradphis(np,np,2)   ! grad phi at the surface, computed once in model initialization
+
+    real (kind=real_kind) :: mubottom(np,np)                       !mu
```
[the remark this answers is not in the document]
Comment (oksanaguba) [reply components/homme/src/theta-l/element_state.F90]: why? mubottom is obtained in caar and not changed in dirk, or by remap. or does it get changed by remap?

**Comment by oksanaguba** [comment]
Homme suite is ok, it is ready to be reviewed again @mt5555 .

**Comment by oksanaguba** [comment]
Assigning to myself since it was approved, unless there are objections.

**Comment by rljacob** [comment]
@oksanaguba please merge this.

**Comment by oksanaguba** [comment]
Merged to next.

**Pull request merged**